lib/yaml_db/serialization_helper.rb

```diff
@@ -195,7 +196,17 @@ def self.table_record_count(table)
         ActiveRecord::Base.connection.select_one("SELECT COUNT(*) FROM #{Utils.quote_table(table)}").values.first.to_i
       end
 
-    end
+      # Just return the first column as sort key unless both columns are "id
+      # columns", in which case it joins them for ordering
+      def self.sort_key(table)
+        key, second_column = table_column_names(table)
+
+        # If this looks like a standard HABTM join table, add the second "id
+        # column" of the join table for ordering
+        key = [key, second_column] if [key, second_column].all? { |name| name =~ /_id$/ }
```

Comment:
I think I'd prefer to see this take the form
```
if condition
  both columns
else
  just the one
end
```
rather than do reassignment and then require an explicit return.